packages/formation/js/accordion.js

```diff
@@ -16,60 +16,63 @@ const isElementInViewport = (
 };
 
 const loadAccordionHandler = () => {
-  const usaAccordion = [
-    ...document.getElementsByClassName('usa-accordion'),
-    ...document.getElementsByClassName('usa-accordion-bordered'),
-  ];
-  const usaAccordionContentElements = [
-    ...document.getElementsByClassName('usa-accordion-content'),
-  ];
+  document
+    .querySelectorAll('.usa-accordion-content:not([aria-hidden])')
+    .forEach(el => {
+      if (el.getAttribute('aria-hidden') === null) {
+        const buttonElement = el.parentElement.querySelector(
+          `[aria-controls="${el.id}"]`,
+        );
+        const hiddenValue = !toBoolean(
+          buttonElement.getAttribute('aria-expanded'),
+        );
 
-  usaAccordionContentElements.forEach(el => {
-    if (el.getAttribute('aria-hidden') === null) {
-      el.setAttribute(
-        'aria-hidden',
-        !toBoolean(el.previousElementSibling.getAttribute('aria-expanded')),
-      );
-    }
-  });
+        el.setAttribute('aria-hidden', hiddenValue);
+      }
+    });
 
-  usaAccordion.forEach(element => {
-    element.addEventListener('click', e => {
-      const target = e.target;
-      const other = Array.from(
-        element.getElementsByClassName('usa-accordion-button'),
-      ).filter(item => item !== target);
+  document
+    .querySelectorAll('.usa-accordion, .usa-accordion-bordered')
```

Comment:
This is a lot better then using the spread operator. I didn't know querySelectorAll returned an element that had a forEach method in it. Thank @rianfowler